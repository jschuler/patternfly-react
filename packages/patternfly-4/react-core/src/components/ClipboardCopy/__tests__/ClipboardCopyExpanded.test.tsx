// Auto-generated do not edit

import * as React from 'react';
import { shallow } from 'enzyme';
import { ClipboardCopyExpanded } from '../ClipboardCopyExpanded';

describe('ClipboardCopyExpanded test', () => {
  it('ClipboardCopyExpanded should match snapshot', () => {
    const view = shallow(
      <ClipboardCopyExpanded
        className={"''"}
				children={<div>ReactNode</div>}
				onChange={(): any => undefined}
				isReadOnly={false}
				isCode={false}
      />);
    expect(view).toMatchSnapshot();
  });
});
