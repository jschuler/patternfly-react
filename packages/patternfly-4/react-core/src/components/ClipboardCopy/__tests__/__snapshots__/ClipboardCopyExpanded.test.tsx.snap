// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClipboardCopyExpanded test ClipboardCopyExpanded should match snapshot 1`] = `
<div
  className="pf-c-clipboard-copy__expandable-content ''"
  contentEditable={true}
  onInput={[Function]}
  suppressContentEditableWarning={true}
>
  <div>
    ReactNode
  </div>
</div>
`;
